### PR TITLE
Feature/combobox

### PR DESCRIPTION
**Pull request by dramoturg**

### CODE REVIEW

File: src/components/Combobox/Combobox.jsx [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+		const className = classnames(theme.container, {
+			[theme.container_isDisabled]: isDisabled
+		});
+
+		return (
+			<div className={className}>
+				<div className={theme.selectbox}>
+					<Selectbox {...selectboxProps} />
+				</div>
+				<Input {...inputProps}/>
```
Comment (raveclassic) [inline src/components/Combobox/Combobox.jsx]: You should not overlap Anchor with Input but compose them in one container

File: src/components/Combobox/Combobox.jsx [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+import classnames from 'classnames';
+
+export const COMBOBOX = Symbol('Combobox');
+
+@PURE
+@themr(COMBOBOX)
+export default class Combobox extends React.Component {
+
+	static propTypes = {
+		children: React.PropTypes.node,
```
Comment (raveclassic) [inline src/components/Combobox/Combobox.jsx]: all these propTypes can be reused from Selectbox

File: src/components/Selectbox/SelectboxAnchor.jsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+	text: React.PropTypes.string,
+	content: React.PropTypes.string,
+	content_hasCaret: React.PropTypes.string,
+	caret: React.PropTypes.string,
+	caret_isReversed: React.PropTypes.string
+};
+
+@PURE
+@themr(SELECTBOX_ANCHOR)
+export default class SelectboxAnchor extends React.Component {
```
Comment (raveclassic) [inline src/components/Selectbox/SelectboxAnchor.jsx]: why don't you use it in Selectbox?

Comment (dramoturg) [inline src/components/Selectbox/SelectboxAnchor.jsx]: Сustom anchor is not used. This component must be removed.

File: src/components/Combobox/Combobox.jsx (hunk 1 above)
Comment (dramoturg) [inline src/components/Combobox/Combobox.jsx]: Popover calculate position relative anchor. If anchor be less than container in which it located input,  popover will have wrong position.

**Pull request merged**